dmscripts/send_dos_opportunities_email.py

```diff
@@ -8,12 +8,40 @@
 logger = logging.getLogger('script')
 
 
-def get_live_briefs_between_two_dates(data_api_client, lot_slug, start_date, end_date, framework_slug):
+MAILCHIMP_LIST_IDS = {
+    'digital-outcomes-and-specialists': {
+        'digital-specialists': None,
+        'digital-outcomes': None,
+        'user-research-participants': None,
+    },
+    'digital-outcomes-and-specialists-2': {
+        'digital-specialists': "30ba9fdf39",
+        'digital-outcomes': "97952fee38",
+        'user-research-participants': "e6b93a3bce",
+    },
+    'digital-outcomes-and-specialists-3': {
+        'digital-specialists': "bee802d641",
+        'digital-outcomes': "5c92c78a78",
+        'user-research-participants': "34ebe0bffa",
+    },
+    'digital-outcomes-and-specialists-4': {
+        'digital-specialists': "29d06d5201",
+        'digital-outcomes': "4360debc5a",
+        'user-research-participants': "2538f8a0f1",
+    }
+}
+
+LOT_NAMES = {
+    "digital-specialists": "Digital specialists",
+    "digital-outcomes": "Digital outcomes",
+    "user-research-participants": "User research participants",
+}
+
+
+def get_live_briefs_between_two_dates(data_api_client, start_date, end_date):
     """Get all briefs for a lot which were published between 2 dates."""
```

Comment:
We should update the docstring to reflect the change in behaviour

```suggestion
    """Get all briefs which were published between 2 dates."""
```